docs/index.md

```diff
@@ -1,31 +1,35 @@
-# Welcome
+# NGINX Ingress Controller
```

Comment:
Alternatives:
- Leave `# Welcome`
- Switch to `# Overview`

Reply:
This will revise the heading on https://kubernetes.github.io/ingress-nginx/

Reply:
I like # Overview